services/order.py

```diff
@@ -0,0 +1,31 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str | None = None
+) -> list[dict]:
+    user = User.objects.get(username=username)
```

Comment:
still not fixed. you should use get_user_model() function from django to get the auth class from settings

Reply:
still not fixed. User get_user_model function instead of explicitly calling auth user model